### PR TITLE
Adding licence check hooks to travis

### PR DESCRIPTION
**Pull request by afein**
We initially discussed having the Kubedash project without a Makefile and follow CAdvisor's deployment process.

However, as Kubedash is not just a Go package, a Makefile could prove useful for container building, sanity checks and eventual UI testing frameworks

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	go get github.com/tools/godep
+
+build: clean deps sanitize
+	GOOS=linux GOARCH=amd64 GCO_ENABLED=0 godep go build -a .
+
+clean: 
+	rm -f kubedash
+
+container: build
+		docker build -t $(PREFIX)/heapster:$(TAG) .
```
Comment (vishh) [inline Makefile]: update image name.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+all: build
+TAG = latest
```
Comment (vishh) [inline Makefile]: Lets try not to push against latest.

**Comment by afein** [comment]
addressed comments, will squash after an LGTM

**Comment by vishh** [comment]
LGTM

**Comment by vishh** [comment]
Feel free to self merge after rebasing

**Comment by afein** [comment]
self-merging

**Pull request merged**